docs/fast_data/configuration/import_export.md

```diff
@@ -0,0 +1,125 @@
+
+:::info
+At the moment, nested imported resources are not exported. e.g. when exporting an ER schema imported by reference which contains a collection imported by reference as well, the collection will not be exported. 
+:::
+
+## Import
+
+After clicking on the `Import` button, it will be possible to upload a `.json` file containing exported data inside a modal.
+
+The imported file should follow the format of files downloaded using the `Export` feature.
+
+If the file is valid, information about the source project, branch and exporting time will be displayed.
+
+![Import modal with uploaded file](./img/fast_data_ie_import_file.png)
+
+When clicking on the `Next` button, all of the Fast Data resources contained inside the imported file will be displayed in their respective tabs. It will be possible to select and deselect resources before completing the import process by clicking on the `Confirm and import` button.
+
+![Import modal with resources selection](./img/fast_data_ie_import_select.png)
+
+Fast Data resources can be imported in two ways:
+
+1) as a reference: the resources will be referenced inside the current project configuration and displayed in read-only mode. It will be possible to access their data but the resources will not be editable. A `Reference` label will be added to their name inside the Fast Data section: when hovering it, a popover will display information about the exporting source project, branch and time.
+
+2) as a resource: the resources will be copied inside the current project configuration and displayed as if they were created directly inside the current project. It will be possible to access and edit their data.
+
+In both cases, an error message will be displayed if some of the imported resources have the same name of resources or references already existing inside the project current configuration.
+
+![Import modal with error](./img/fast_data_ie_import_error.png)
+
+It will be possible to update references only by importing resources with the same name and from the same source project. In this case, the `Reference update` label will be displayed alongside each updated resource and its content will be overwritten inside the configuration.
+
+![Import modal with reference update](./img/fast_data_ie_import_update.png)
+
+:::caution
+In both cases, microservices associated to each resource (such as Real-Time Updaters and their Message Formats, Single View Creators or Single View Trigger Generators) will NOT be imported. Please make sure these services have been created before deploying your configuration.
+:::
+
+:::info
+Some Fast Data resources, such as Projections, can be imported as a reference only in projects that are using the Single View Trigger Generator architecture. Please check your services before importing the resources.
+:::
+
+## FAQs on Import / Export
```

Comment:
@fedepini Spostiamolo nella sezione FAQ e qui mettiamo solamente una domanda del tipo: When should I use the Import / Export functionality? To learn more see the proper section (e ci linki la pagina delle FAQ)